integration/config.ts

```diff
@@ -6,11 +6,12 @@ LoggerInstance.setLevel(LogLevel.Error)
 
 const configBase: Config = {
     nodeUri: 'http://localhost:8545',
-    metadataUri: 'http://nevermined-metadata:5000',
+    metadataUri: 'http://172.17.0.1:5000',
```

Comment:
This is the equivalent of `host.docker.internal` for linux, but not sure if it's going to work for Mac & windows or WSL

Reply:
Maybe a conditional statement here would be great? IDK, up to you

Reply:
I can just revert that